docs/repositories.md

```diff
@@ -16,15 +16,15 @@ rules_ts_dependencies(<a href="#rules_ts_dependencies-ts_version_from">ts_versio
 
 Dependencies needed by users of rules_ts.
 
-To skip fetching the typescript package, define repository called `npm_typescript` before calling this.
+To skip fetching the typescript package, call `rules_ts_bazel_dependencies` instead.
```

Comment:
I don't think this should be getting updated? This is also the internal one?

Reply:
The new function introduced here really is public API. It's better than the suggestion I'm removing here, just like the case in this repo, you might want to install rules_ts bazel dependencies, then declare `@npm`, then load from it, so it's not nice to require that you create an `npm_typescript` repo before installing the bazel deps.